[2024-05-03T15:45:07.145+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-03T15:45:07.151+0530] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_api_data.print_another_message manual__2024-05-03T10:09:56.831201+00:00 [queued]>
[2024-05-03T15:45:07.154+0530] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_api_data.print_another_message manual__2024-05-03T10:09:56.831201+00:00 [queued]>
[2024-05-03T15:45:07.154+0530] {taskinstance.py:2303} INFO - Starting attempt 1 of 3
[2024-05-03T15:45:07.161+0530] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): print_another_message> on 2024-05-03 10:09:56.831201+00:00
[2024-05-03T15:45:07.163+0530] {standard_task_runner.py:63} INFO - Started process 84314 to run task
[2024-05-03T15:45:07.164+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_api_data', 'print_another_message', 'manual__2024-05-03T10:09:56.831201+00:00', '--job-id', '943', '--raw', '--subdir', 'DAGS_FOLDER/async_api_dag.py', '--cfg-path', '/tmp/tmpsqi6x1is']
[2024-05-03T15:45:07.165+0530] {standard_task_runner.py:91} INFO - Job 943: Subtask print_another_message
[2024-05-03T15:45:07.179+0530] {task_command.py:426} INFO - Running <TaskInstance: fetch_api_data.print_another_message manual__2024-05-03T10:09:56.831201+00:00 [running]> on host ninebit5lm
[2024-05-03T15:45:07.208+0530] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fetch_api_data' AIRFLOW_CTX_TASK_ID='print_another_message' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T10:09:56.831201+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-03T10:09:56.831201+00:00'
[2024-05-03T15:45:07.208+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-03T15:45:07.209+0530] {logging_mixin.py:188} INFO - Another message!
[2024-05-03T15:45:07.209+0530] {python.py:237} INFO - Done. Returned value was: None
[2024-05-03T15:45:07.209+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-03T15:45:07.210+0530] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=fetch_api_data, task_id=print_another_message, execution_date=20240503T100956, start_date=20240503T101507, end_date=20240503T101507
[2024-05-03T15:45:07.256+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-03T15:45:07.267+0530] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-03T15:45:07.267+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
